ci-operator/step-registry/assisted/vsphere/teardown/machine/assisted-vsphere-teardown-machine-commands.sh

```diff
@@ -12,9 +12,14 @@ if [[ -z "${LEASED_RESOURCE}" ]]; then
   exit 1
 fi
 
-mkdir -p /home/assisted-test-infra/build/terraform
-cp ${SHARED_DIR}/terraform.tgz /home/assisted-test-infra/build/terraform
-cd /home/assisted-test-infra/build/terraform
-tar -xvf terraform.tgz
-terraform init
-terraform destroy -force
\ No newline at end of file
+if [[ -f "${SHARED_DIR}/terraform.tgz" ]]; then
+  mkdir -p /home/assisted-test-infra/build/terraform
+  cp ${SHARED_DIR}/terraform.tgz /home/assisted-test-infra/build/terraform
+  cd /home/assisted-test-infra/build/terraform
+  tar -xvf terraform.tgz
+  terraform init -input=false
+  terraform apply -destroy -input=false -auto-approve
+else
+  echo "Terraform file does not exist - probably websphere machine does not created - skipping teardown"
```

Comment:
... probably websphere machine hasn't been created...

Reply:
And let's short-circuit this code branch, meaning:
if <file does not exist>
  echo...
  exit 0

...